doc/source/user_guide/indexing.rst

```diff
@@ -584,7 +584,7 @@ IX indexer is deprecated
 
    The ``.ix`` indexer was removed, in favor of the more strict ``.iloc`` and ``.loc`` indexers.
 
-``.ix`` offers a lot of magic on the inference of what the user wants to do. To wit, ``.ix`` can decide
+``.ix`` offers a lot of magic on the inference of what the user wants to do. To with, ``.ix`` can decide
```

Comment:
can you revert the `to wit` change (here and in the other file), cf https://github.com/pandas-dev/pandas/issues/37252#issuecomment-712275889